PROJECT

```diff
@@ -0,0 +1,11 @@
+domain: openshift.io
```

Comment:
You split `updateservice.operator.openshift.io` into an `updateservice.operator` group and an `openshift.io` domain, but the SDK docs say:

> The domain is everything after the first DNS segment. Using cache.example.com as an example, the --domain would be example.com.

So I suspect we want an `updateservice` group, and then either `operator.openshift.io` as the domain, or just `openshift.io` (`updateservice` seems sufficiently specific without the `operator` scoping).

[1]: https://v1-0-x.sdk.operatorframework.io/docs/building-operators/golang/project_migration_guide/#create-a-new-project

Reply:
I based the group name on what we decided on in https://issues.redhat.com/browse/OTA-258 which was  updateservice.operator. I specifically tested this out first with a generate as well before committing to it and it did cause any issues.